pkg/resource/plugin/analyzer_plugin.go

```diff
@@ -259,21 +262,36 @@ func (a *analyzer) GetAnalyzerInfo() (AnalyzerInfo, error) {
 		return AnalyzerInfo{}, rpcError
 	}
 
-	policies := []apitype.Policy{}
+	policies := []AnalyzerPolicyInfo{}
 	for _, p := range resp.GetPolicies() {
 		enforcementLevel, err := convertEnforcementLevel(p.EnforcementLevel)
 		if err != nil {
 			return AnalyzerInfo{}, err
 		}
 
-		policies = append(policies, apitype.Policy{
+		schema := convertConfigSchema(p.GetConfigSchema())
+
+		// Inject `enforcementLevel` into the schema.
+		if schema != nil {
```

Comment:
> Do we want to support configurable enforcement levels even if there is no specific configuration for the policy? I would think yes, but then I imagine it may be hard to distinguish btw a policy pack that supports config but has no config and a policy pack that does not support config.

Yes, we definitely want configurable enforcement levels even if there is no config schema for a policy. I removed the nil check and am now always injecting enforcementLevel into the schema, even if the policy did not have it's own config schema defined.

From the service's perspective, is not seeing any schema for any of the policies how it is determining whether a policy pack is configurable?

If so, I'll need some other way in the CLI to determine if the policy pack supports config. I could add a new property `supportsConfig` to the metadata it gets back over the GRPC interface, which would default to false for older policy packs. Or add a new `SupportsConfig()` RPC that could be called to check. Or, since we haven't released a new `@pulumi/policy` that uses the version tag, I could check the version property you added. If that's a non-empty string, then the policy pack supports config. Leaning towards this latter approach.

Reply:
I went ahead and added another commit that adds the `supportsConfig` bool property to the metadata returned from the policy pack. For older policy packs, this will come back as the default value of `false`, in which case we do not send any config schema for such policy packs so the service can distinguish whether a policy pack is configurable or not based on whether it has config schema.